### PR TITLE
Software Library Updates

### PR DESCRIPTION
**Pull request by DivyaSreeMunagavalasa**

### CODE REVIEW

**Comment by ThorodanBrom** [comment]
ok to test
________________________________

**Comment by ThorodanBrom** [comment]
retest this please

File: pom.xml
Comment (ThorodanBrom) [inline pom.xml:1]: This version needs to match the Keycloak version used.

Comment (DivyaSreeMunagavalasa) [reply pom.xml:1]: yes, it is matching.

**Comment by ThorodanBrom** [comment]
retest this please

**Comment by ThorodanBrom** [comment]
> Commit [eda60e1](https://github.com/datakaveri/iudx-aaa-server/pull/343/commits/eda60e1f9af61db28cbe4147fa3de006265daab7) needs to be reverted - Flyway migrations are failing when tests are being run. - https://jenkins.iudx.io/job/iudx%20aaa-server%20(PR)%20pipeline/487/

For the flyway dependency, the following changes need to be done as per this issue from the flyway **if we are using the latest version of the `flyway-core` dependency (> version 10)**
- Include the latest `flyway-core` dependency
- Include a new dependency of `org.flywaydb:flyway-database-postgresql` (https://central.sonatype.com/artifact/org.flywaydb/flyway-database-postgresql)

This probably needs to be done for every server that uses Flyway specifically for PostgreSQL and the `flyway-core` version has been bumped to > v10.

**Comment by ThorodanBrom** [comment]
@datakaveri/devops there is a port bind error on post 8443 before running integration tests that's preventing the integration tests from passing - https://jenkins.iudx.io/view/iudx-PR/job/iudx%20aaa-server%20(PR)%20pipeline/489/console

**Comment by karun-singh** [comment]
> @datakaveri/devops there is a port bind error on post 8443 before running integration tests that's preventing the integration tests from passing - https://jenkins.iudx.io/view/iudx-PR/job/iudx%20aaa-server%20(PR)%20pipeline/489/console

@ThorodanBrom resolved now, it was due to an aborted pipeline run, Container did not get cleaned.

**Comment by ThorodanBrom** [comment]
> > @datakaveri/devops there is a port bind error on post 8443 before running integration tests that's preventing the integration tests from passing - https://jenkins.iudx.io/view/iudx-PR/job/iudx%20aaa-server%20(PR)%20pipeline/489/console
> 
> @ThorodanBrom resolved now, it was due to an aborted pipeline run, Container did not get cleaned.

Thanks @karun-singh

**Comment by ThorodanBrom** [comment]
retest this please

**Comment by ThorodanBrom** [comment]
@Kranthi-Guribilli noted that the Vert.x PgPool are deprecated in the version of Vert.x we are using. We may need to update all instances of `PgPool` to [Pool](https://vertx.io/docs/apidocs/io/vertx/sqlclient/Pool.html), and build said pool using [PgBuilder](https://vertx.io/docs/apidocs/io/vertx/rxjava3/pgclient/PgBuilder.html)

**Review by ThorodanBrom — APPROVED** [review]

**Pull request merged**